conf/f2b_jail.conf

```diff
@@ -0,0 +1,6 @@
+[__APP__]
+enabled = true
+port = http,https
+filter = __APP__
+logpath = /var/log/__APP__/nextcloud.log
+maxretry = 5
```

Comment:
```suggestion
maxretry = 5
bantime = 86400
findtime = 43200
```

Reply:
We could, but I thought we could let the default value set by YNH